extensions/interactions/rule_templates.json

```diff
@@ -236,5 +236,21 @@
     "FuzzyEquals": {
       "description": "is equal to {{x|NormalizedString}}, misspelled by at most one character"
     }
+  },
+  "RulesHtmlMapping": {
```

Comment:
This is conceptually incorrect. Each key here so far represents an interaction, and this is not an interaction. You should make a separate json file for it.

Reply:
done